.github/workflows/unit-tests.yml

```diff
@@ -3,13 +3,11 @@ on: [push, pull_request]
 jobs:
   container-test-job:
     runs-on: ubuntu-latest
-    container: maierbn/opendihu:latest
+    container: carmehp/opendihu-ci
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-        with:
-          path: opendihu
-      - name: Build 
-        working-directory: opendihu
-        run: python3 dependencies/scons/scons.py BUILD_TYPE=RELEASE no_examples=TRUE
+      - name: Checkout and rebuild
+        run: |
+          cd /workspace/opendihu
+          git pull origin ${{ github.event.pull_request.head.sha }}
```

Comment:
pretty sure this just broke forks as the origin in the container always points to https://github.com/opendihu/opendihu

Reply:
This is true, but at the same time this is only the temporary hacky approach. See the discussion here: https://github.com/opendihu/opendihu/issues/18

Reply:
I look at the workflow file again, and I found that this does not work for push, since `github.event.pull_request.head.sha` does not exists in this case. Hence I changed it to `on: [pull_request]` .
The fork issue remains.